src/ensembl/src/content/app/species-selector/components/popular-species-button/PopularSpeciesButton.tsx

```diff
@@ -55,7 +56,14 @@ export const PopularSpeciesButton = (props: Props) => {
 
     if (!is_available) {
       return;
-    } else if (isSelected) {
+    }
+
+    analyticsTracking.setCustomDimension({
+      diemension: CustomDimensions.SPECIES,
+      value: genome_id
+    });
```

Comment:
Again, perhaps something to move to redux middleware?

Reply:
Created separate setter functions for dimensions as discussed.